packages/pluggableWidgets/pie-doughnut-chart-native/src/PieDoughnutChart.tsx

```diff
@@ -0,0 +1,24 @@
+import { createElement, ReactElement } from "react";
+import { all } from "deepmerge";
+
+import { PieDoughnutChartProps } from "../typings/PieDoughnutChartProps";
+import { PieDoughnutChart as PieDoughnutChartComponent } from "./components/PieDoughnutChart";
+import { ChartStyle, defaultStyle } from "./ui/Styles";
+import { useSeries } from "./utils/SeriesLoader";
+
+export function PieDoughnutChart(props: PieDoughnutChartProps<ChartStyle>): ReactElement | null {
+    const { series: chartSeries, style, ...rest } = props;
+    const series = useSeries(chartSeries);
+
+    if (!series) {
+        return null;
+    }
```

Comment:
Can you move this check to `PieDoughnutChart.editorConfig` so Studio Pro throws an error when no series are configured?

Reply:
I don't think this is required; in the widget's XML it declares at least one data source is required `(required=true` is default in XML). 

This check is useful for when any of the series' X or Y attribute value's are not available.

Do you think it can stay?

Reply:
Ah, not sure if it will give you the expected behavoir then. Because we don't actually check the X / Y attributes, we just check if `series` is not `null` or `undefined`

Reply:
@wesleysieses pushed a change, please re-review